CHANGELOG.md

```diff
@@ -13,6 +13,10 @@
 
 * **[Fix]** Fix crash involving SDK's `ms_viewWillAppear` method.
 
+### App Center Crashes
+
+* **[Behavior Change]** `MSCrashesDelegate` callback methods are now invoked on the main thread (`willSendErrorReport`,  `didSucceedSendingErrorReport`, and `didFailSendingErrorReport`).
```

Comment:
inconsistent casing of Change vs change in this file.

Reply:
```suggestion
* **[Behavior change]** `MSCrashesDelegate` callback methods are now invoked on the main thread (`willSendErrorReport:`,  `didSucceedSendingErrorReport:`, and `didFailSendingErrorReport:`).
```